app/screens/home/search/search.tsx

```diff
@@ -149,6 +149,7 @@ const SearchScreen = ({teamId, teams}: Props) => {
 
     const handleClearSearch = useCallback(() => {
         clearRef.current = true;
+        Keyboard.dismiss();
```

Comment:
I see this may be important, but I don't see how it handles the original issue.

Reply:
This is the line that solves the issue, if the keyboard was not dismissed the styles rendering the header were calculated as if the keyboard was still opened when the search was cleared causing the bug.